web-app/spec/messages/messages_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails/all'
+require_relative '../../app/modules/messages_module'
+
+module MessagesModule
+    MessagesModule.number_cel_from = '+17066230189'
+  class TestClient
+  end
+
+  context "Message module" do
+
+    it "send sms message" do
+      number = "+525530265963"
+      msg = "Hello World"
+      client = TestClient.new
+      allow(MessagesModule).to receive(:twilio_connection) { client }
+      allow(client).to receive(:create).with(hash_including(from: '+17066230189', to: number, body: msg)){sms_response}
+      expect(client).to receive(:create).with(hash_including(from: '+17066230189', to: number, body: msg))
+      MessagesModule.send_SMS_message(number,msg)
+    end
+
+    def sms_response
```

Comment:
Move this to a `let(sms_response) { {... your hash ...} }` statement just below `context`.